resteasy/src/main/java/com/cerner/beadledom/resteasy/ModuleProcessor.java

```diff
@@ -0,0 +1,80 @@
+package com.cerner.beadledom.resteasy;
+
+import com.google.inject.Binding;
```

Comment:
Another option available would be to update `com.cerner.beadledom.jaxrs.provider.FilteringJacksonJsonProvider` class and add  the `@Priority(a number < 5000)` annotation. RESTEasy uses the priority to sort the providers [1]. FilteringJacksonJsonProvider is only used server-side so this would be a safe change to make.

[1] 
https://github.com/resteasy/Resteasy/blob/3.6/resteasy-jaxrs/src/main/java/org/jboss/resteasy/spi/ResteasyProviderFactory.java#L2176-L2177
https://github.com/resteasy/Resteasy/blob/3.6/resteasy-jaxrs/src/main/java/org/jboss/resteasy/spi/ResteasyProviderFactory.java#L1723-L1725